soh/soh/Enhancements/randomizer/ShuffleGrass.cpp

```diff
@@ -0,0 +1,113 @@
+#include "ShuffleGrass.h"
+#include "soh_assets.h"
+#include "static_data.h"
+
+extern "C" {
+#include "variables.h"
+#include "overlays/actors/ovl_En_Kusa/z_en_kusa.h"
+#include "objects/gameplay_field_keep/gameplay_field_keep.h"
+#include "objects/object_kusa/object_kusa.h"
+extern PlayState* gPlayState;
```

Comment:
```suggestion
extern PlayState *gPlayState;
```

Reply:
We don't do this in other places of the code either.